go-metro.yaml.example

```diff
@@ -1,14 +1,38 @@
+# NOTE: for go metro to run unprivileged, you will have to set CAP_NET_RAW capabilities on the binary:
+#        (debian based) $ sudo apt-get update libcap
+#        (redhat based) $ sudo yum install compat-libcap1
+#        (redhat based - alternatively) $ sudo yum install compat-libcap1
+#        $ sudo setcap cap_net_raw+ep /opt/datadog-agent/bin/go-metro
+
 init_config:
-    snaplen: 512
-    idle_ttl: 300
-    exp_ttl: 60
+    snaplen: 512            # should be >=104 (to accomodate for the largest possible TCP header)
+    idle_ttl: 300           # time after which an idle flow (no traffic received) is flushed.
+    exp_ttl: 60             # time after which a finished flow is flushed.
     statsd_ip: 127.0.0.1
     statsd_port: 8125
     log_to_file: true
-    log_level: debug
+    log_level: info         # available levels: trace, debug, info, warning, error, critical
 
 config:
-- interface: en0
-  tags: [mytag]
-  ips: []
-  hosts: [myhost]
+- interface: eth0           # metrics will be also tagged by interface.
+  tags:
+    - foo
```

Comment:
could we rather have a `foo:bar` tag as an example (good practice to use key:value metrics instead of keys only) :)